### PR TITLE
workflow 7.x | port workflow metrics and update for 7.3 | LRDOCS-7461

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-7461

### CODE REVIEW

File: docs/dxp/7.x/en/process-automation/workflow/user-guide/using-workflow-metrics.md [hunk 1]
```diff
@@ -76,6 +72,30 @@ The Pause field is if there's an event in the workflow when time should stop cou
 | Days | Enter a whole number for number of days. If there are partial days such as 36 hours, use in combination with the _Hours_ field; express it 1 Day and 12 Hours. |
 | Hours | Enter the number of hours and minutes. The values in the _Hours_ box must not exceed _23:59_; if the duration exceeds one day, use in combination with the _Days_ field. |
 
+## Configuring the SLA Job Interval
+
+The _Check SLA Job Interval_ is the time between metrics recalculation. The default values is 1 minute. To change it,
```
[the remark this answers is not in the document]
Comment (rbohl) [reply docs/dxp/7.x/en/process-automation/workflow/user-guide/using-workflow-metrics.md:77]: @matheus-xavier what about

> The time interval between metrics recalculation is called the _SLA Job Interval_. The default values is 1 minute. To change it, ...

File: docs/dxp/7.x/en/process-automation/workflow/user-guide/workflow-metrics-reports.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+![In this view, the only process with pending items is the Auto Insurance Application.](./workflow-metrics-reports/images/06.png)
+
+A table view of all installed workflow processes shows you how many items are overdue, how many are on time, and how many are pending in the workflow process.
+
+Access more detailed reports by clicking on one of the workflow processes.
+
+The Reports UI has two main views, represented as tabs: _Dashboard_ and _Performance_.
+
+The Dashboard view is displayed by default. Included are items currently in the workflow process, even those untracked by the SLA. Untracked items include those in the paused step of the workflow and items outside the scope of the configured SLA duration.
```
[the remark this answers is not in the document]
Comment (rbohl) [reply docs/dxp/7.x/en/process-automation/workflow/user-guide/workflow-metrics-reports.md:13]: Wat about

> The Dashboard view is displayed by default. Included are items currently in the workflow process, even those untracked by the SLA. Untracked items include those that pre-date the SLA itself or those that are in the process but have not reached the step where the SLA is triggered.

File: docs/dxp/7.x/en/process-automation/workflow/user-guide/using-workflow-metrics.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (matheus-xavier) [reply docs/dxp/7.x/en/process-automation/workflow/user-guide/using-workflow-metrics.md]: Looks good!

File: docs/dxp/7.x/en/process-automation/workflow/user-guide/workflow-metrics-reports.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (matheus-xavier) [reply docs/dxp/7.x/en/process-automation/workflow/user-guide/workflow-metrics-reports.md]: It's awesome!

**Pull request closed without merging**